packages/manager/modules/pci/src/projects/project/activate/activate.routing.js

```diff
@@ -112,6 +112,9 @@ export default /* @ngInject */ ($stateProvider) => {
   registerPCINewPaymentState($stateProvider, {
     stateName: paymentStateName,
     configStep: false,
+    bindings: {
+      discoveryPromotionVoucherAmount: '@',
+    },
```

Comment:
```suggestion
```

No need in routing, you should put in component file.

Reply:
Currently, this bindings is needed : 
The current implementation of the activate router (not yet reviewed) contains a nested state that will initialise another component (the voucher component).

To keep this review in it's scope, It's possible to require the  creation of a sub-component instead of having a nested state when this draft PR https://github.com/ovh/manager/pull/10554 will be reviewed.